src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -198,4 +198,39 @@ export default {
       },
     };
   },
+  addressValidationSuccess(address, _options) {
```

Comment:
is `_options` being used and/or should it have a default value?

Reply:
Nope needs to be removed.. I wasn't sure if I was going to need that or not.  Using `_` will consider it an optional value and won't throw a linter error if its not used.

Reply:
Awesome I didn't know that trick!